interface/billing/sl_eob_invoice.php

```diff
@@ -284,10 +284,10 @@ function updateFields(payField, adjField, balField, coPayField, isFirstProcCode)
 $patient_id = 0 + $ferow['pid'];
 $encounter_id = 0 + $ferow['encounter'];
 $svcdate = substr($ferow['date'], 0, 10);
-$form_payer_id = ($_POST['$form_payer_id']) ? (0 + $_POST['form_payer_id']) : 0;
+$form_payer_id = ($_POST['form_payer_id']) ? (0 + $_POST['form_payer_id']) : 0;
 $form_reference = $_POST['form_reference'];
-$form_check_date = ($_POST['form_check_date']) ? DateToYYYYMMDD($_POST['form_check_date']) : date('Y-m-d');
-$form_deposit_date = ($_POST['form_deposit_date']) ? DateToYYYYMMDD($_POST['form_deposit_date']) : date('Y-m-d');
+$form_check_date   = fixDate($_POST['form_check_date'], date('Y-m-d'));
+$form_deposit_date = fixDate($_POST['form_deposit_date'], $form_check_date);
```

Comment:
this is odd, but keep it here (I noted you needed it on the patch and definitely ok for a quick fix to deal with some mangled date by DateToYYYYMMDD). I'll put this on my list of things to look into since this is a core function (guessing the input going into it is not right).

Reply:
nope date input was fine. It is already correctly formatted going to function but gets spit out -yyyy-mm-dd-

Reply:
doesn't sounds good since this is used in lots of places. will check out the function.

Reply:
i'll do it Brady as i've got it queued up.

Reply:
The place to start with is what the "Date Display Format" setting is in Globals. I am guessing the input is not matching that set format.

Reply:
I checked out the function and if the input format doesn't match correctly then it would shoot back -yyyy-mm-dd- (if Date Display Format is not yyyy-mm-dd)

Reply:
the input is set to use the date display format (`$datetimepicker_formatInput = true;`), which can confirm that the date is in the correct format when use the date picker

I above is ok, then am suspicious that maybe a incorrectly formatted date is grabbed here:
```
f2.form_check_date.value        = f1.form_paydate.value;
```

Reply:
yep that's my case as CCM dates require mm/dd/yyyy and won't follow Globals date. Just something else.


> f2.form_check_date.value        = f1.form_paydate.value;

Is yyyy-mm-dd doesn't need a fixup and comes from datepicker.

Reply:
bringing in so will be on demo reset. I do know one datepicker needs to be reconfigured to honor the Global but is only one place.(payment posting container)